content/en/docs/contribute/_index.md

```diff
@@ -95,17 +95,17 @@ class A,B,C,D,E,F,G,H,M,Q,N,O,P,V grey
 class S,T,U spacewhite
 class first,second,third white
 {{</ mermaid >}}
-***Figure - Getting started for a new contributor***
+Figure 1. Getting started for a new contributor.
```

Comment:
Does Mermaid provide a way to label a figure?

Reply:
Has been discussed per [add title to graph issue](https://github.com/mermaid-js/mermaid/issues/1433) but nothing implemented at this time.